tools/docker/tests/test_flows.sh

```diff
@@ -664,5 +681,8 @@ main() {
 VERBOSE=false
 SKIP_INIT=false
 STOP_ON_FAILURE=false
+DUMP=false
+DUMPFILE=-
+TCPDUMP_PROC=
```

Comment:
You don't really need to set a default value for `TCPDUMP_PROC` since it's either unused or set by the script. Same for `DUMPFILE`.

(nitpick)

Reply:
Also why is the default output file stdout? I would think that there are other stuff on stdout that would prevent it to be used correctly, but I didn't try.

Reply:
Ok, changed